### PR TITLE
Update Release Notes for Taipy GUI 2.1.0 (#356)

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #356.

### CODE REVIEW

File: docs/relnotes.md [hunk 1]
```diff
@@ -18,16 +18,40 @@ Published on 2023-01.
...
 
 
-#### New Features
+### New Features
 
-**`taipy-gui`**<br/>2.1.0
+<h6 style="font-size: 1.2em"><strong><code>taipy</code></strong></h6>
+2.1
 
+- Taipy and all its dependencies now support Python 3.11.
```
Comment (jrobinAV) [inline docs/relnotes.md]:
```suggestion
- Taipy and all its dependencies now support Python 3.11.
  See [Python documentation](https://docs.python.org/3/whatsnew/3.11.html) for details.
```

**Review by jrobinAV — APPROVED** [review]

File: docs/relnotes.md [hunk 2]
```diff
@@ -18,16 +18,40 @@ Published on 2023-01.
...
+  [this section](manuals/gui/configuration.md#protect-your-application-files) for
+  details.
+- Charts can use a [Decimator]() instance that cleverly filters data
+  points out to greatly improve performance.<br/>
+  See the [Decimator documentation]() for more details.
 - Charts now support polar, funnel, candlesticks and many other types of charts.<br/>
-  See the [chart control](../manuals/gui/viselements/chart) section for details.
+  See the [chart control](manuals/gui/viselements/chart.md) section for details.
+- Charts now support the dark theme automatically.
+- Tooltips can be set on individual cells.<br/>
```
Comment (FredLL-Avaiga) [inline docs/relnotes.md]: Cells -> table cells? Or mention that this is related to tables

File: docs/relnotes.md [hunk 3]
```diff
@@ -18,16 +18,40 @@ Published on 2023-01.
...
+  See the [example](manuals/gui/viselements/table.md#cell-tooltip) for more information.
+- [Long running callbacks](manuals/gui/callbacks.md#long-running-callbacks) have
+  been improved to allow for easily returning a value.<br/>
+  See the documentation of the `invoke_long_callback()^` function or the
+  [issue #547](https://github.com/Avaiga/taipy-gui/issues/547) for more details.
 - Developers can specify the location of the Taipy webapp, for debugging purposes.<br/>
-  Use the `--webapp-path` command line option to use that.
+  The `--webapp-path` command line option allows to specific that location.<br/>
```
Comment (FredLL-Avaiga) [inline docs/relnotes.md]: Specify?

File: docs/relnotes.md [hunk 4]
```diff
@@ -110,18 +147,22 @@ Published on 2022-10.
...
 
-#### Improvements and changes
+### Improvements and changes
+
+<h6 style="font-size: 1.2em"><strong><code>taipy-gui</code></strong></h6>
+2.0.0
 
-**`taipy-gui`**<br/>2.0.0
 - Stopping then re-running the `Gui^` instance is no longer required in Notebook contexts.
 - A discrete graphical indicator is displayed on top of pages when the server is processing.
```
Comment (FredLL-Avaiga) [inline docs/relnotes.md]: Bottom

**Review by FredLL-Avaiga — APPROVED** [review]

File: docs/relnotes.md (hunk 2 above)
Comment (FabienLelaquais) [reply docs/relnotes.md]: 👍

File: docs/relnotes.md (hunk 4 above)
Comment (FabienLelaquais) [reply docs/relnotes.md]: 👍

File: docs/relnotes.md (hunk 1 above)
Comment (FabienLelaquais) [reply docs/relnotes.md]: Done.

**Review by dinhlongviolin1 — APPROVED** [review]
Looks good to me!

**Pull request merged**